examples/landing_zone/main.tf

```diff
@@ -0,0 +1,65 @@
+##############################################################################
+# Workload Resource Group
+##############################################################################
+
+module "workload_resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.0.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.workload_resource_group == null ? "${var.prefix}-workload-rg" : null
+  existing_resource_group_name = var.workload_resource_group
+}
+
+module "management_resource_group" {
+  source = "git::https://github.com/terraform-ibm-modules/terraform-ibm-resource-group.git?ref=v1.0.5"
+  # if an existing resource group is not set (null) create a new one using prefix
+  resource_group_name          = var.management_resource_group == null ? "${var.prefix}-management-rg" : null
+  existing_resource_group_name = var.management_resource_group
+}
+
+#############################################################################
+# Provision cloud object storage and bucket
+#############################################################################
+
+resource "ibm_resource_instance" "cos_instance" {
+  count = var.enable_vpc_flow_logs ? 1 : 0
+
+  name              = "${var.prefix}-vpc-logs-cos"
+  resource_group_id = module.workload_resource_group.resource_group_id
+  service           = "cloud-object-storage"
+  plan              = var.cos_plan
+  location          = var.cos_location
+}
+
+resource "ibm_cos_bucket" "cos_bucket" {
```

Comment:
Could you use https://github.com/terraform-ibm-modules/terraform-ibm-cos/tree/main/profiles/fscloud here to create a fscloud compliant bucket

Reply:
FYI, this profile provisions two cos buckets and enables replication between them. The cos buckets need to be in different regions too

Reply:
@vburckhardt ,
if we have to use this fscloud compliant bucket, we would need activity_tracker, sysdig, two key protects each for primary and secondary buckets. This would make the example very much complicated. Should I go ahead and add all those services in the example?

Reply:
@Aashiq-J - Agree - let's just add a comment in the README for the example (starting that the cos instance/bucket is not fscloud compliant in this example), and just use the base [https://github.com/terraform-ibm-modules/terraform-ibm-cos](https://github.com/terraform-ibm-modules/terraform-ibm-cos) module then without at, sysdig, key protect.